server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -320,6 +336,52 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
         this.clearOpMaxTimeTimer();
 
         this.removeAllListeners();
+
+        this.logSessionEndMetrics(closeType);
+    }
+
+    private logSessionStartMetrics() {
+        if (this.sessionStartMetric?.isCompleted()) {
+            this.sessionStartMetric = this.createSessionMetric();
+        }
+
+        if (this.isNewDocument) {
+            if (this.scribeStartSuccess) {
+                this.sessionStartMetric?.setProperties({ [CommonProperties.sessionState]: SessionState.started });
+                this.sessionStartMetric?.success("Session started successfully");
+            } else {
+                this.sessionStartMetric?.setProperties({ [CommonProperties.sessionState]:
+                    SessionState.scribeLambdaDown });
+                this.sessionStartMetric?.error("Scribe lambda failed");
+            }
+        } else {
+            this.sessionStartMetric?.setProperties({ [CommonProperties.sessionState]: SessionState.resumed });
```

Comment:
When we are resuming a session (so not a new document) is it possible that `this.scribeStartSuccess` is `false`, meaning the session failed to resume?

Reply:
Fixed. Please take a look. I thought of addressing the session resume cases in a sperate PR as it involves adding the lambda details to the checkpoint but I have now added it to this PR itself.